inference-engine/tests/ie_test_utils/functional_test_utils/layer_test_utils.cpp

```diff
@@ -22,6 +40,30 @@ void LayerTestsCommon::Run() {
     Validate();
 }
 
+void LayerTestsCommon::Serialize() {
+    SKIP_IF_CURRENT_TEST_IS_DISABLED();
+
+    std::string out_xml_path = test_name() + "_" + timestamp() + ".xml";
+    std::string out_bin_path = test_name() + "_" + timestamp() + ".bin";
+
+    ngraph::pass::Manager manager;
+    manager.register_pass<ngraph::pass::Serialize>(out_xml_path, out_bin_path);
+    manager.run_passes(function);
+
+    InferenceEngine::Core ie;
+    auto result = ie.ReadNetwork(out_xml_path, out_bin_path);
+
+    bool success;
+    std::string message;
+    std::tie(success, message) =
+        compare_functions(result.getFunction(), function);
+
+    ASSERT_TRUE(success) << message;
```

Comment:
I think ASSERT_* macros stop the test execution if the condition is false. This means that the files will not be deleted in case this happens. I suggest switching to EXPECT_* instead.

Reply:
In general gtest tests architecture propose to use `TearDown` method to clean-up artifacts after a test. Maybe it will be more convenient and obvious